web-client/iron-remote-gui/package.json

```diff
@@ -1,38 +1,47 @@
 {
-  "name": "@devolutions/iron-remote-gui",
-  "author": "Nicolas Girot",
-  "email": "ngirot@devolutions.net",
-  "contributors": [
-    "Benoit Cortier",
-    "Zacharia Ellaham"
-  ],
-  "description": "Web Component providing agnostic implementation for Iron Wasm base client.",
-  "version": "0.8.1",
-  "type": "module",
-  "private": true,
-  "scripts": {
-    "dev": "npm run pre-build && vite",
-    "build": "npm run pre-build && vite build",
-    "build-alone": "vite build",
-    "pre-build": "node ./pre-build.js",
-    "preview": "vite preview",
-    "check": "svelte-check --tsconfig ./tsconfig.json"
-  },
-  "devDependencies": {
-    "@sveltejs/vite-plugin-svelte": "^2.0.0",
-    "@tsconfig/svelte": "^3.0.0",
-    "rxjs": "^6.6.7",
-    "svelte": "^3.54.0",
-    "svelte-check": "^2.10.0",
-    "tslib": "^2.4.1",
-    "typescript": "^4.9.3",
-    "vite": "^4.0.0",
-    "vite-plugin-dts": "^1.7.1",
-    "vite-plugin-top-level-await": "^1.2.2",
-    "vite-plugin-wasm": "^3.1.0"
-  },
-  "dependencies": {
-    "@types/ua-parser-js": "^0.7.36",
-    "ua-parser-js": "^1.0.33"
-  }
+    "name": "@devolutions/iron-remote-gui",
+    "author": "Nicolas Girot",
+    "email": "ngirot@devolutions.net",
+    "contributors": [
+        "Benoit Cortier",
+        "Zacharia Ellaham"
+    ],
+    "description": "Web Component providing agnostic implementation for Iron Wasm base client.",
+    "version": "0.8.1",
+    "type": "module",
+    "private": true,
+    "scripts": {
+        "dev": "npm run pre-build && vite",
+        "build": "npm run pre-build && vite build",
+        "build-alone": "vite build",
+        "pre-build": "node ./pre-build.js",
+        "preview": "vite preview",
+        "check": "svelte-check --tsconfig ./tsconfig.json",
+        "lint": "prettier . --check . && eslint src/**",
+        "format": "prettier . --write ."
+    },
+    "devDependencies": {
+        "@sveltejs/vite-plugin-svelte": "^2.0.0",
+        "@tsconfig/svelte": "^3.0.0",
+        "@typescript-eslint/eslint-plugin": "^6.9.1",
+        "eslint": "^8.52.0",
+        "eslint-config-prettier": "^9.0.0",
+        "eslint-plugin-prettier": "^5.0.1",
+        "eslint-plugin-svelte": "^2.34.1",
+        "prettier": "^3.0.3",
+        "prettier-plugin-svelte": "^3.0.3",
+        "rxjs": "^6.6.7",
+        "svelte": "^3.54.0",
+        "svelte-check": "^2.10.0",
+        "tslib": "^2.4.1",
+        "typescript": "^4.9.3",
```

Comment:
Svelte known error, does not yet support typescript over 5.0
https://github.com/sveltejs/kit/issues/8650